assets/js/components/settings/SettingsRenderer.js

```diff
@@ -30,8 +30,8 @@ const { useSelect, useDispatch } = Data;
 
 export default function SettingsRenderer( { slug, isOpen, isEditing } ) {
 	const storeName = `modules/${ slug }`;
```

Comment:
This still needs to change to use the registered store name.

Reply:
@benbowler you marked this as resolved but the comment is not yet addressed. As an aside, please don't mark feedback as resolved as it makes it harder to see (possibly how you missed it in this case as well).

Reply:
[Addressed here](https://github.com/google/site-kit-wp/pull/2525/files#diff-c7a8015e4e1004ea9e43de2e0a06e5b64345dedb0dac21dc94ef5a449969ac7eR32-R34)